### DISCUSSION TITLE
Which ubuntu versions are supported exactly?

### DISCUSSION THREAD

**Discussion by mariusfilipowski** [runs-on/runs-on#110] [Q&A] [answered]
Since I did not find more than an example for ubuntu22: Is ubuntu20 and 24 also supported, as in GitHub?

**Comment by crohr** [comment, marked as answer]
Currently, only ubuntu22 is supported. ubuntu20 won't be supported. ubuntu24 will be supported soon, just waiting a bit for official images to stabilise.
